packages/common/core-interfaces/Removing-IFluidRouter.md

```diff
@@ -90,7 +90,36 @@ const entryPoint = await container.getEntryPoint();
 
 ### Aliased DataStores
 
-(Not yet written)
+The `getRootDataStore` API has been deprecated on `IContainerRuntime` and `ContainerRuntime`. Migrate to usage of the new `IContainerRuntime.getAliasedDataStoreEntryPoint(...)` endpoint.
+The following is an example of that this change may look like:
+
+```ts
+// OLD
+const alias = "alias";
+const dataStore = await containerRuntime.createDataStore(TestDataObjectType);
+await dataStore.trySetAlias(alias);
+
+const retrievedDataStore = await containerRuntime.getRootDataStore(alias);
+```
+
+```ts
+// NEW
+const alias = "alias";
+const dataStore = await containerRuntime.createDataStore(TestDataObjectType);
+await dataStore.trySetAlias(alias);
+
+const entryPoint: IFluidHandle<ITestDataObject> | undefined =
+	await containerRuntime.getRootDataStore(alias);
+const retrievedDataStore = await entryPoint?.get();
+```
+
+### Sub-object routing
```

Comment:
I am wondering if we want to even mention sub-object routing. We should eventually replace the request there with some other function like `resolveHandle` or `resolveChildObject`. Maybe it's worth keeping this document until then since it is supported.